src/pages/proposal/detail/index.tsx

```
@@ -210,6 +221,21 @@ const Detail = () => {
                 }
                 sx={{ margin: 0 }}
               />
+              {proposalType?.name === ProposalTypes.General && (
+                <Typography
+                  variant="h6"
+                  dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)